src/styles/blocks/button.scss

```diff
@@ -0,0 +1,45 @@
+.button {
+  display: block;
+  width: 166px;
+  height: 40px;
+  margin-left: 17px;
+  margin-top: 16px;
+  border: none;
+  border-radius: 5px;
+  background-color: #00acdc;
+
+  &:hover {
+    display: block;
+    width: 166px;
+    height: 40px;
+    margin-left: 17px;
+    margin-top: 16px;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    background-color: #ffffff;
+  }
+}
+
+.buy {
+  display: block;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-decoration: none;
+  color: #ffffff;
+
+  &:hover {
+    display: block;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 700;
+    font-style: normal;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    color: #00acdc;
```

Comment:
Similar to the previous comment, you should remove the repeated properties inside the &:hover selector for the .buy class. Only specify the properties that change when the element is hovered over.